cookbooks/fb_systemd/recipes/default.rb

```diff
@@ -128,6 +128,13 @@
   mode '0755'
 end
 
-link '/etc/systemd/system/default.target' do
-  to lazy { node['fb_systemd']['default_target'] }
+execute 'set default target' do
+  not_if do
+    current = Mixlib::ShellOut.new('systemctl get-default').run_command.
+      stdout.strip
+    current != node['fb_systemd']['default_target']
```

Comment:
Shouldn't we not do this when the current is the same as the default_target?  You have it backwards.